data/toc.yaml

```diff
@@ -1,58 +1,84 @@
-Guides:
+Getting started:
```

Comment:
Should this be "Get started" to match the top-nav name when the breadcrumbs are generated.
```suggestion
Get started:
```